docs/docs/features/backlight.md

```diff
@@ -6,7 +6,7 @@ sidebar_label: Backlight
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-Backlight is a feature used to control array of LEDs, usually placed through or under switches. Unlike [RGB Underglow](underglow.md), backlight currently allows only one color per LED, also LEDs are not addressable, so you can't control individual LEDs.
+Backlight is a feature used to control an array of LEDs, usually placed through or under switches. Unlike [RGB Underglow](underglow.md), backlight can only control single color LEDs. Additionally, because backlight LEDs all receive the same power, it's impossible to dim individual LEDs.
```

Comment:
I think it would be nice to take this opportunity to separate out this information from the feature’s introduction:

```suggestion
Backlight is a feature used to control an array of LEDs, usually placed through or under switches. 

:::info
Unlike [RGB Underglow](underglow.md), backlight can only control single color LEDs. Additionally, because backlight LEDs all receive the same power, it's not possible to dim individual LEDs.
:::
```

In that suggestion I also swapped “impossible” for “not possible” which feels a little more friendly — but that’s purely subjective. 🙂

Reply:
I'm happy with that, friendlier is always better!